internal/testacc/project_environment_resource_test.go

```diff
@@ -0,0 +1,99 @@
+package testAcc
```

Comment:
This entire test file is commented out, is this intentional?

Reply:
Yes for both environment and imports.

Both are the moment incompatiable to test with terraform. Need modification for both